src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+public class Ui {
```

Comment:
Since this class is a user interface, you might consider allocate all the input/output tasks to it (this might also correspond to my previous comment about functional abstraction).